azureadps-2.0-preview/AzureAD/Get-AzureADDirectoryRole.yml

```diff
@@ -31,7 +31,7 @@ examples:
 
     ObjectId                             DisplayName                        Description
     --------                             -----------                        -----------
-    019ea7a2-1613-47c9-81cb-20ba35b1ae48 Company Administrator              Company Administrator role has full access to perform any operation in the company scope.
+    aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb Company Administrator              Company Administrator role has full access to perform any operation in the company scope.
```

Comment:
Not all Ids have been updated here.

Reply:
As above, these look like they should be in an allow list, these have the potential to break things if removed.